crates/bvs-vault-router/src/msg.rs

```diff
@@ -126,6 +131,11 @@ pub enum QueryMsg {
 
     #[returns(SlashingRequestResponse)]
     SlashingRequest(SlashingRequestId),
+
+    #[returns(SlashLockedResponse)]
+    SlashLocked {
```

Comment:
should we keep the name to slashing?
```suggestion
    #[returns(SlashingLockedResponse)]
    SlashingLocked {
```